### ISSUE TITLE
Were webhooks added to core?

### ISSUE THREAD

**Issue by janewilkin** [mitcho/hookpress#9]
Hi Mitcho,

I'm checking out your plugin because I'm needing a way to export actions to a django app. 

Have webhooks been added to wordpress core? 

I found this support article that seemed to suggest that it's default behavior on wordpress.com, but I guess not in open-source wordpress?

https://en.support.wordpress.com/webhooks/

**Comment by mitcho** [comment]
You're right, that's wordpress.com, not dotorg. It is, however, based on this plugin.
